modules/gapi/test/common/gapi_imgproc_tests_inl.hpp

```diff
@@ -625,6 +625,41 @@ TEST_P(YUV2BGRTest, AccuracyTest)
         EXPECT_EQ(out_mat_gapi.size(), std::get<1>(param));
     }
 }
+
+TEST_P(NormalizeTest, Test)
+{
+    auto param = GetParam();
+
+    compare_f cmpF = std::get<0>(param);
+    const auto in_type = std::get<1>(param);
+    const auto in_size = std::get<2>(param);
+    const auto a = std::get<3>(param);
+    const auto b = std::get<4>(param);
+    const auto norm_type = std::get<5>(param);
+    const auto ddepth = std::get<6>(param);
+    const auto createOut = std::get<7>(param);
+    auto compile_args = std::get<8>(param);
+    int dtype = CV_MAKETYPE(ddepth, CV_MAT_CN(in_type));
```

Comment:
What about `std::tie()`?

If a new parameter is either added or removed, you'd not need to fix all indexes in every `std::get<>`